info/v1/container.go

```diff
@@ -415,6 +415,9 @@ type FsStats struct {
 	// Number of bytes available for non-root user.
 	Available uint64 `json:"available"`
 
+	// HasInodes when true, indicates that Inodes info will be available.
+	HasInodes bool `json:"has_inodes"`
```

Comment:
trying to think how to handle this for container spec:

* zfs, devicemapper do not support inodes at all
* btrfs, xfs have inode limits that can dynamically grow

not sure where in code we want to populate the `FsSpec`